aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1107,6 +1116,24 @@ module aptos_framework::stake {
         assert!(exists<StakePool>(pool_address), error::invalid_argument(ESTAKE_POOL_DOES_NOT_EXIST));
     }
 
+    fun validate_network_and_fullnode_addresses(network_addresses: vector<u8>, fullnode_addresses: vector<u8>) {
```

Comment:
I'd also call this validate_validator_and_fullnode_addresses

Reply:
Updated thanks